doc/developer/zebra.rst

```diff
@@ -234,221 +234,219 @@ Zebra Protocol Commands
 +-----------------------------------------+-------+
 | ZEBRA_VRF_LABEL                         | 35    |
 +-----------------------------------------+-------+
-| ZEBRA_INTERFACE_VRF_UPDATE              | 36    |
```

Comment:
oof - yeah, I'd forgotten about this table. I think it's not worth documenting these values, and maintaining this table. Let's just refer to the header file?